docusaurus/docs/dev-docs/quick-start.md

```diff
@@ -166,71 +183,55 @@ You have just created a basic data structure for your Strapi project! You can ke
+Now that your beautiful first Strapi project is working locally, it's time for the world to see it live! The most straightforward way to host your project is to use Strapi Cloud: Deploying your project on Strapi Cloud is done with a single command! 🚀
 
-<details style={detailsStyle}>
-<summary style={summaryStyle}>Step 1: Host the code of your Strapi project on GitHub</summary>
+To deploy your project on Strapi Cloud, in your terminal:
 
-### Step 1: Host the code of your Strapi project on GitHub
+1. If the server for your local Strapi project is running, which should be the case if you followed this tutorial so far, press `Ctrl-C` to stop the server.
+2. Ensure you are in the folder of your Strapi project (if needed, run for instance `cd my-strapi-project` to reach this folder), and run the following command:
 
-Create a new GitHub repository and push the code of your Strapi project to this repository. If you're not already familiar with GitHub, the togglable content below should get you started 👇
+    <Tabs groupId="yarn-npm">
 
-<details>
-<summary>Steps required to push your Strapi project code to GitHub:</summary>
+    <TabItem value="yarn" label="Yarn">
 
-1. In the terminal, ensure you are still in the `my-project` folder that hosts the Strapi project we created. If you followed this tutorial closely so far, we should still be there.
-2. Run the `git init` command to initialize git for this folder.
-3. Run the `git add .` command to add all modified files to the git index.
-4. Run the `git commit -m "Initial commit"` command to create a commit with all the added changes.
-5. Log in into your GitHub account and [create a new repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/quickstart-for-repositories). Give the new repository a name, for instance `my-first-strapi-project`, and remember this name.
-6. Go back to the terminal and push your local repository to GitHub:
+      ```sh
+      yarn strapi deploy
+      ```
 
-  a. Run a command similar to the following: `git remote add origin git@github.com:yourname/my-first-strapi-project.git`, ensuring you replace `yourname` by your actual GitHub profile name, and `my-first-strapi-project` by the actual name you used at step 4.
+    </TabItem>
 
-  b. Run the `git push --set-upstream origin main` command to finally push the commit to your GitHub repository.
+    <TabItem value="npm" label="NPM">
 
-Additional information about using git with the command line interface can be found in the [official GitHub documentation](https://docs.github.com/en/migrations/importing-source-code/using-the-command-line-to-import-source-code/adding-locally-hosted-code-to-github#adding-a-local-repository-to-github-using-git).
+      ```sh
+      npx @strapi/strapi deploy
```

Comment:
Could you please double-check the commands for me? My draft was done using the QA/experimental builds and commands differ. 🙏

Reply:
@nathan-pichon

Reply:
LGTM

Reply:
OK, @nathan-pichon. 
By "released version" do you mean the CMS-related part?

The Quick Start Guide needs to be absolutely 100% correct when Strapi 'releases' the feature and the docs.

Reply:
Yes I mean CMS part.
It is how it's supposed to look (either yarn strapi deploy, strapi deploy or npx @strapi/strapi deploy), but before the release of the CMS, it will always be something like `@strapi/strapi@experimental-0.0.0.198236496492` or the path to the local file of the CLI.

Reply:
Ah yes, got it! My question was not clear enough, indeed, sorry for that 😅

I just wanted to know whether it will be run by `npx` (and not with `npm run`) and on which package.

I guess it's also possible we have a `@latest` "tag" maybe? (i.e. `npx run @strapi/strapi@latest`)? 🤔

So, to sum up, commands are:
- for yarn: `yarn strapi deploy`
- for NPM: 
   - "local" run: `npm run strapi deploy` 
   - or direct execution by npx as in `npx @strapi/strapi deploy`
- `strapi deploy` for people who already installed the strapi CLI globally (which [we don't recommend](https://docs.strapi.io/dev-docs/cli), though, but who knows… 🤷 )

Is that correct or should we remove one of the 4 options?

Reply:
Yes @latest should be used I guess; as if they didn't download the latest version of strapi, they won't have the cloud part.

Reply:
And yes, all the commands you have listed should work.